### ISSUE TITLE
Characters overlapping / glitched rendering of specific line of text

### ISSUE THREAD

**Issue by DFranzkoch** [notepad-plus-plus/notepad-plus-plus#10895]
### Description of the Issue
Characters overlapping / glitched rendering of specific line of text

### Steps to Reproduce the Issue
Copy/pase the following line of text into a new file

localcli --plugin-dir /usr/lib/vmware/esxcli/int/ deviceInternal alias store --bus-type logical --alias vmnic0 --bus-address "pci#s00000008.03#0"

This will not render correctly. 0 and s characters after "pci#" are overlapping
The line is grabbed from a VMWare Knowledgebase article found here:
https://kb.vmware.com/s/article/2091560

Searching for pci#s00000008.03 gets a result. So the internal value of the string is as it should. This seems to be purely a font rendering issue.

I am unable to produce screenshots of the strange line, as the screenshots I grab with Greenshot are rendered correctly. WTF??

I tried changing the font, problem persists.

### Expected Behavior
Each character should be visible individually

### Actual Behavior
Characters overlap each other

### Debug Information
Notepad++ v8.1.9.2   (32-bit)
Build time : Nov 21 2021 - 04:27:12
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows Server 2008 R2 Datacenter (64-bit) 
OS Build : 7601.0
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll mimeTools.dll NppConverter.dll NppExport.dll NPPJSONViewer.dll XMLTools.dll

**Comment by vortaj** [comment]
Maybe use a camera then?

**Comment by DFranzkoch** [comment]
I investigated further and found the problem. The broken characters have been caused by RDP compression / hardware acceleration on my bugged Linux Remote Desktop client (based on Igel Linux / Ubuntu). I failed to mention that the broken string was shown inside a Remote Desktop Session. Didn't think it was relevant at the time. It was.

When conneted to the same session from a Windows 10 computer, the font is rendered correctly. When connected from an Igel Thinclient running Igel OS v11.05.133 the problem occurs. This is why screenshotting was not possible - on the host machine the rendering was correct internally, which was shown by the captured screenshot. The mangling of the characters took place when transmitting the screen to the thinclient.
I managed to capture the broken rendering by using the screenshot tool running on the thinclient itself and Greenshot inside the RDP session.
![KKXzEWXHe8](https://user-images.githubusercontent.com/88775776/145534686-27242bb1-2720-43ca-b667-68587e09ee48.png)
.
